lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -2571,8 +2571,8 @@ def test_report_fails_if_error(self, data_collector_module, upload_func):
             with patch('lms.djangoapps.instructor_task.tasks_helper.runner._get_current_task') as mock_current_task:
                 mock_current_task.return_value = self.current_task
 
-                response = upload_func(None, None, self.course.id, None, 'generated')
-                self.assertEqual(response, UPDATE_STATUS_FAILED)
+                response = upload_ora2_data(None, None, self.course.id, None, 'generated')
```

Comment:
why was this changed? This method needs to test the upload_func provided by ddt above.

Reply:
Rebased on master and fixed the conflicts